{
  try {
    sourceAndContext.second.computeErrors(sourceAndContext.first);
    return sourceAndContext.second;
  }
 catch (  AnalysisException e) {
    LOG.info(e);
  }
  return null;
}
